ice_setup/ice.py

```diff
@@ -669,7 +674,7 @@ def sync(cls, repos, distro):
                         '--newest-only',
                         '--norepopath',
                         '-p',
-                        destination
+                        destination % repo_directories[repo]
                     ]
```

Comment:
This whole section from the line with ``repo_directories = ...`` on down needs a bit more rework.

The backstory is that previously we were taking both the mon and osd channels, and syncing them  *both* into a repo named ``ceph``.  We don't need to do that anymore.  If you look at the ``repo_mapping`` dict, it only has one key, ``ceph``.  The idea was to make a dictionary that essentially listed all of the CDN sources that needed to be sync'd to that repo, and be flexible enough to handle different names for different versions of RHEL.

If we were going to keep that same scheme, the ``repo_mapping`` would become:
```
        repo_mapping = {
            'ceph-osd': {
                'sources': {
                    '6': ['rhel-6-server-rhceph-1.3-osd-rpms'],
                    '7': ['rhel-7-server-rhceph-1.3-osd-rpms']
                },
                'destination': '/opt/calamari/webapp/content/OSD'
            },
            'ceph-mon': {
                'sources': {
                    '6': ['rhel-6-server-rhceph-1.3-mon-rpms'],
                    '7': ['rhel-7-server-rhceph-1.3-mon-rpms']
                },
                'destination': '/opt/calamari/webapp/content/MON'
            },
        }
```

Hopefully that makes more sense.

As written, it would still be trying to combine everything in the sources dict into the repo.  It would also throw a key error when it hit ``destination = repo_mapping[repo]['destination']`` because ``repo`` is one of ``ceph-mon`` or ``ceph-osd``, but the dict only has key of ``ceph``.

If you take the repo mapping as I have it, you can also get rid of the ``repo_directories`` dict.